Bowling.java

```diff
@@ -117,6 +113,18 @@ public Integer getScore() {
 		return totalPoints;
 	}
 
+	public Boolean isGameOver() {
+		currentFrame = getCurrentFrame();
+		secondToLastFrame = getFrameBefore(currentFrame);
+		if (secondToLastFrame.isPresent()) {
+			thirdToLastFrame = getFrameBefore(secondToLastFrame.get());
+		}
+		
+		return allFrames.size() == 11 && 
+			   !secondToLastFrame.get().isStrike() && 
+			   !secondToLastFrame.get().isSpare();
```

Comment:
Given that the game should only ever have ten frames I'm not sure why it matters if the previous frame is a spare or strike. Was there an existing test that required that test in order to pass?

More bluntly - was that the simplest code that would've passed the test you were given?